src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -634,23 +510,19 @@ private String returnPermissionsProviders() {
 			// Try Vault
 			test = getServer().getPluginManager().getPlugin("Vault");
 			if (test != null) {
-				RegisteredServiceProvider<Chat> chatProvider = findChatImplementation();
-				RegisteredServiceProvider<Permission> permissionProvider = getServer().getServicesManager().getRegistration(Permission.class);
-
-				if (chatProvider == null) {
+				net.milkbowl.vault.chat.Chat chat = getServer().getServicesManager().load(net.milkbowl.vault.chat.Chat.class);
```

Comment:
Why was this undone? `findChatImplementation()` is an important bandaid for plugins that break things for us and others.

Reply:
should be back